mcapi/src/internal/java/net/flintmc/mcapi/internal/render/text/DefaultFontRenderBuilder.java

```diff
@@ -145,21 +149,48 @@ public FontRenderBuilder disableShadow() {
     return this;
   }
 
+  /** {@inheritDoc} */
+  @Override
+  public FontRenderBuilder matrixStack(Object matrixStack) {
+    this.matrixStack = matrixStack;
+    return this;
+  }
+
   /** {@inheritDoc} */
   @Override
   public void draw() {
     this.validate();
 
-    this.renderer.drawString(
-        this.x,
-        this.y,
-        this.text,
-        this.rgba,
-        this.alignment,
-        this.maxLineLength,
-        this.shadow,
-        this.xFactor,
-        this.yFactor);
+    boolean isNetherUpdate = this.versionHelper.isUnder(16);
```

Comment:
Personally I think a better name for this variable would be helpful, such as `hasRGBText` (which I assume is what is new, text-wise, unless I'm wrong)

Reply:
I have renamed it.